ui/src/app/pages/admin/vwb/admin-vwb-workspace.tsx

```diff
@@ -3,16 +3,45 @@ import { useEffect, useState } from 'react';
 import { Column } from 'primereact/column';
 import { DataTable } from 'primereact/datatable';
 
-import { VwbWorkspace } from 'generated/fetch';
+import { VwbWorkspace, VwbWorkspaceSearchParamType } from 'generated/fetch';
 
 import { Button } from 'app/components/buttons';
-import { Error, TextInputWithLabel } from 'app/components/inputs';
+import { styles as headerStyles } from 'app/components/headers';
+import { Error, Select, TextInputWithLabel } from 'app/components/inputs';
 import { SpinnerOverlay } from 'app/components/spinners';
 import { WithSpinnerOverlayProps } from 'app/components/with-spinner-overlay';
 import { vwbWorkspaceAdminApi } from 'app/services/swagger-fetch-clients';
 
+const VwbWorkspaceSearchParamTypeOptions = [
+  {
+    value: VwbWorkspaceSearchParamType.CREATOR,
+    label: 'Workspace Creator',
+  },
+  {
+    value: VwbWorkspaceSearchParamType.ID,
+    label: 'Workspace ID',
```

Comment:
there are 2 IDs, one is UUID, another one is called user-facing ID. 
I think what support want to use is the user facing ID

Reply:
Workspace User Facing Id will be less confusing, and we might also want to add UUID in the future because that is easier for eng to debug

Reply:
Makes sense. Update the enum and UI labels